docs/source/class_basics.rst

```diff
@@ -321,3 +321,17 @@ class, including an abstract method defined in an abstract base class.
 
 You can implement an abstract property using either a normal
 property or an instance variable.
+
+An abstract class that defines at least one abstract method or
```

Comment:
I would rather put this at the end of https://github.com/python/mypy/blob/master/docs/source/final_attrs.rst#final-classes